plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotConnectorSmokeTest.java

```diff
@@ -907,10 +907,10 @@ public void testShowCreateTable()
         assertThat((String) computeScalar("SHOW CREATE TABLE region"))
                 .isEqualTo(
                         "CREATE TABLE %s.%s.region (\n" +
-                                "   regionkey bigint,\n" +
-                                "   updated_at_seconds bigint,\n" +
+                                "   comment varchar,\n" +
                                 "   name varchar,\n" +
-                                "   comment varchar\n" +
+                                "   regionkey bigint,\n" +
+                                "   updated_at_seconds bigint\n" +
```

Comment:
Do you know why the column order is changed? Is it possible to restore the original behavior?

Reply:
Seems not possible. Columns are sorted